tests/robustness/multiclient_test/framework/harness.go

```diff
@@ -145,6 +149,16 @@ func (th *TestHarness) getSnapshotter() bool {
 		return false
 	}
 
+	// Set size limits for content cache and metadata cache for repository under test.
```

Comment:
Can the cache limits be set in `ConnectOrCreateRepo` ?

Reply:
I thought about it. Doing so will require changes to the function signature, i.e. passing the cache set flags. I decided not to do it in order to avoid fallout from the function being used in other places.